pkg/crc/services/dns/dns.go

```diff
@@ -20,9 +20,8 @@ import (
 
 const (
 	dnsServicePort    = 53
-	dnsContainerIP    = "10.88.0.8"
 	publicDNSQueryURI = "quay.io"
-	crcDnsmasqService = "crc-dnsmasq.service"
+	dnsmasqService    = "dnsmasq.service"
```

Comment:
This service is already installed/available in the VM?

Reply:
yes it is already installed and available in the VM